redis/src/main/scala/zio/redis/RespValue.scala

```diff
@@ -0,0 +1,197 @@
+package zio.redis
+
+import java.nio.charset.StandardCharsets
+
+import zio.stream.{ Sink, ZSink }
+import zio.{ Chunk, IO, Ref }
+
+sealed trait RespValue extends Any {
```

Comment:
You should be able to remove the extends part.

Reply:
This lets the case classes extend `AnyVal`. I'm not 100% sure how practically useful that is, but it might allow them to be put into chunks without boxing.